pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -53,6 +53,21 @@ import (
 const (
 	// DefaultBootstrapTokenTTL default ttl for bootstrap token
 	DefaultBootstrapTokenTTL = 10 * time.Minute
+
+	// MachineRegionAnnotationName as annotation name for a machine region
+	MachineRegionAnnotationName = "machine.openshift.io/region"
+
+	// MachineAZAnnotationName as annotation name for a machine AZ
+	MachineAZAnnotationName = "machine.openshift.io/availability-zone"
+
+	// MachineInstanceIDAnnotationName as annotation name for a machine instance ID
```

Comment:
this one is not really needed, consumers can look at `machine.Spec.ProviderID`

Reply:
Removed